cmd/geth/snapshot.go

```diff
@@ -337,18 +337,37 @@ func pruneBlock(ctx *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	var newAncientPath string
-	oldAncientPath := ctx.GlobalString(utils.AncientFlag.Name)
-	if !filepath.IsAbs(oldAncientPath) {
-		// force absolute paths, which often fail due to the splicing of relative paths
-		return errors.New("datadir.ancient not abs path")
+
+	// Most of the problems reported by many users when using the prune-block
```

Comment:
These comments can be placed in the description tips of pr,  not suitable as code comments

Reply:
yes, the log clearly states that an absolute path is required, otherwise the execution will not continue, which can already be used by the correct user

Reply:
Agree